recipes/diffusioncma/meta.yaml

```
@@ -0,0 +1,62 @@
+{% set name = "diffusioncma" %}
+{% set version = "0.5.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.com/Olllom/diffusioncma/-/archive/{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0d94957efb08f212557ae625f304c7d8f2aa2e68e40b65776385f45a1fa6290e
+
+build:
+  number: 0
+  skip: True  # [win or py<35]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake >=3.12.1 
+    - libgomp  # [not osx]
+    - libtool  # [unix]
+    - llvm-openmp  # [osx]
+
+  host:
+    - python
+    - eigen
+    - pybind11
+    - numpy
+    - setuptools
+    - libcmaes >=0.9.6
+
+  run:
+    - python
+    - Click >=6.0
```

Dependencies are all lowercase.
```suggestion
    - click >=6.0
```